### PR TITLE
Update installation.md per ID review

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
     
-    a. **UPDATE SITE** 
+    a. Install by using an update site URL. 
 
-    Find the update site URL.  E.g. for the [0.7.0 release](https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.7.0.202212141445), the URL would be: https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/liberty-tools-eclipse/0.7.0/repository/
+    Find the update site URL for your chosen release from the entry in the Releases list. For example, the URL for the [0.7.0 release](https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.7.0.202212141445) is https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/liberty-tools-eclipse/0.7.0/repository/.
 
-    Click on the `Add...` button to open the `Add Repository` view. Specify a name, copy/paste the URL above as the location, and click on the `Add` button. 
+    Click the **Add...** button to open the **Add Repository** view. Specify a name, copy and paste the update site URL as the location, and click the **Add** button.
```
Comment (dmuelle) [inline docs/installation.md:38]: > Specify a name,....

What name? The screenshot shows "Liberty Tools"- is that required or is this a customizable field?

File: docs/installation.md [hunk 2]
```diff
@@ -1,68 +1,68 @@
...
 
     **NOTE:** Do NOT select the "IBM Liberty Developer Tools" selection, which is an earlier, different set of IDE features/plugins, though with the similar name.
 
 ## Using Help -> Install New Software
 
-The Liberty Tools feature can be installed using the artifacts provided in the form of an update site or through an archive.
+You can use the **Install New Software** option in the Eclipse **Help** menu to install Liberty Tools by using an update site or through an archive.
 
-1. Go to menu: `Help`->`Install New Software`.
+1. From the Eclipse menu in your IDE, select **Help** > **Install New Software**.
```
Comment (scottkurz) [inline docs/installation.md:28]: I think "menu bar" is a better fit than "Eclipse menu" so I changed it in my commit.

File: docs/installation.md (hunk 1 above)
Comment (scottkurz) [reply docs/installation.md:38]: addressed in my commit

File: docs/installation.md [hunk 3]
```diff
@@ -1,68 +1,70 @@
...
 
 1. Point your browser to the Liberty Tools marketplace entry: https://marketplace.eclipse.org/content/liberty-tools
-2. Drag the "Install" button to the toolbar of your Eclipse IDE
+2. Drag the **Install** button to the toolbar of your Eclipse IDE
 
-## Using Eclipse Marketplace client
+## Using the Eclipse Marketplace client
 
-1. Go to menu:  `Help`->`Eclipse Marketplace` and type **"Liberty Tools"**.
+1. From the menu bar in your IDE, select **Help** > **Eclipse Marketplace** and type **"Liberty Tools"** and then <kbd>Enter</kbd>.
```
Comment (scottkurz) [inline docs/installation.md:20]: I think "menu bar" is a better fit than "Eclipse menu" so changed it.

File: docs/installation.md [hunk 4]
```diff
@@ -1,68 +1,70 @@
...
 
-    **NOTE:** Do NOT select the "IBM Liberty Developer Tools" selection, which is an earlier, different set of IDE features/plugins, though with the similar name.
+2. Under the **Liberty Tools <X.Y.Z>** entry, click the **Install** button.
```
Comment (scottkurz) [inline docs/installation.md:22]: OK, elaborated on this section since you thought it seemed incomplete.

File: docs/installation.md [hunk 5]
```diff
@@ -1,68 +1,70 @@
...
     
-    a. **UPDATE SITE** 
+    a. Install by using an update site URL. 
 
-    Find the update site URL.  E.g. for the [0.7.0 release](https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.7.0.202212141445), the URL would be: https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/liberty-tools-eclipse/0.7.0/repository/
+    Find the update site URL for your chosen release from the entry in the Releases list. For example, the URL for the [0.7.0 release](https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.7.0.202212141445) is https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/liberty-tools-eclipse/0.7.0/repository/.
 
-    Click on the `Add...` button to open the `Add Repository` view. Specify a name, copy/paste the URL above as the location, and click on the `Add` button. 
+    Click the **Add...** button to open the **Add Repository** view. Copy and paste the update site URL as the **Location**, specify an (optional) **Name** if desired, and click the **Add** button.
```
Comment (scottkurz) [inline docs/installation.md:40]: OK, clarified the "Name" is optional.

**Review by scottkurz — APPROVED** [review]
LGTM

**Pull request merged**